<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkDistanceCalculationBetweenPointsWidget</class>
 <widget class="QWidget" name="QmitkDistanceCalculationBetweenPointsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout"> 
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
        <widget class="QGroupBox" name="ReferenceLandmarksGroupBox">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title">
            <string>Reference points</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="margin">
              <number>6</number>
            </property>
            <item>
              <widget class="QmitkPointListWidget" name="m_ReferencePoints">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QGroupBox" name="m_spaceHolderGroupBox">
                <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                  </sizepolicy>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>37</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>16777215</width>
                    <height>37</height>
                  </size>
                </property>
                <property name="title">
                  <string/>
                </property>
                <property name="alignment">
                  <set>Qt::AlignCenter</set>
                </property>
                <property name="flat">
                  <bool>false</bool>
                </property>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="spacing">
                    <number>0</number>
                  </property>
                  <property name="margin">
                    <number>0</number>
                  </property>
                  <item>
                    <widget class="QPushButton" name="m_AddReferencePositionBtn">
                      <property name="sizePolicy">
                        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                          <horstretch>0</horstretch>
                          <verstretch>0</verstretch>
                        </sizepolicy>
                      </property>
                      <property name="minimumSize">
                        <size>
                          <width>0</width>
                          <height>35</height>
                        </size>
                      </property>
                      <property name="text">
                        <string>&amp;Add reference position</string>
                      </property>
                    </widget>
                  </item>
                </layout>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
     <item>
      <widget class="QGroupBox" name="MeasurementLandmarksGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Measurement points</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QmitkPointListWidget" name="m_MeasurementPoints">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="m_spaceHolderGroupBox2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>37</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>37</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="m_AddMeasurementPositionBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>&amp;Add measurement position</string>
             </property>
            </widget>
           </item>        
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row ="1" column="0">
        <widget class="QGroupBox" name="DistancesGroupBox">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="title">
            <string>Distance values</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="margin">
              <number>6</number>
            </property>
    
            <item>
              <widget class="QListWidget" name="m_Distances">              
              </widget>
            </item>
    
          </layout>
        </widget>
  </item>

    <item row ="1" column="1">
        <layout class="QVBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
            <number>50</number>
          </property>
          <property name="rightMargin">
            <number>50</number>
          </property>
          <item>
            <widget class="QPushButton" name="m_CalculateDistanceBtn">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>0</width>
                  <height>35</height>
                </size>
              </property>
              <property name="text">
                <string>Calculate distance values</string>
              </property>
            </widget>
          </item>
  
          <item>
            <widget class="QPushButton" name="m_SaveDataBtn">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="minimumSize">
                <size>
                  <width>0</width>
                  <height>35</height>
                </size>
              </property>
              <property name="text">
                <string>Save data</string>
              </property>
            </widget>
          </item>
        </layout>
    </item> 
    
    
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QListWidget</extends>
   <header>QmitkPointListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
